src/js/components/Tabs/Tabs.js

```diff
@@ -103,19 +234,154 @@ const Tabs = forwardRef(
         {...rest}
         background={theme.tabs.background}
       >
-        <StyledTabsHeader
-          as={Box}
-          direction="row"
-          justify={justify}
-          alignSelf={alignControls}
-          flex={false}
-          wrap
-          background={theme.tabs.header.background}
-          gap={theme.tabs.gap}
-          {...tabsHeaderStyles}
-        >
-          {tabs}
-        </StyledTabsHeader>
+        <Box direction={overflow ? 'row' : 'column'} {...tabsHeaderStyles}>
```

Comment:
The height once the overflow controls appear is `48px` while the default view when there's enough space for all the Tabs is `38px`. This causes a jump in the UI. Is it possible to eliminate the 10px shift?

Reply:
Are you testing this with the responsive story or with different code? I am getting a height of 50px both when there is enough space for all the tabs and when the tabs overflow and the arrows show up. I am also not seeing a jump in the UI.

Reply:
Oh okay so I was testing with the HPE theme, looks like the height shift is happening in the grommet theme. The reason for this is the left and right arrow buttons have a height of 48px which causes the height of the container to increase when we show the left and right arrows. I can see if I can adjust the left and right buttons.

Reply:
Perfect yes I was testing on the Grommet theme

Reply:
I changed the button arrows to contain an icon wrapped with a box which allowed for more control over the padding around the icon. Let me know if this looks okay